packages/postgres-database/src/simcore_postgres_database/models/resource_tracker.py

```diff
@@ -126,6 +109,23 @@
         nullable=False,
         doc="Service Version (parsed from image label scraped via Prometheus)",
     ),
+    sa.Column(
+        "cpu_limit",
+        sa.Numeric(precision=3, scale=2),
+        nullable=False,
+        doc="CPU resource allocated to a container, ex.0.5 CPU shares",
+    ),
+    sa.Column(
+        "memory_limit",
+        sa.BigInteger,
+        nullable=False,
+        doc="memory limit in bytes scraped via Prometheus",
+    ),
+    sa.Column(
+        "type",
```

Comment:
I try to avoid `type` for name since it is too general and even a reserved word. Less problematic alternatives are `flavor`, `kind`, `classification`

Reply:
i will rename to classification, thank you